query/within.js

```diff
@@ -0,0 +1,33 @@
+const DynamicQueryCache = require('./DynamicQueryCache');
+
+// Remove before PR.
+const logger = require('pelias-logger').get('interpolation');
+
+/**
+  find all streets which have a bbox which envelops the specified point; regardless of their names.
+**/
+
+const SQL = `
+  SELECT street.polyline.id, street.polyline.line, street.names.name FROM street.polyline
+  JOIN street.rtree ON street.rtree.id = street.polyline.id
+  JOIN street.names ON street.names.id = street.polyline.id
+  WHERE ((street.rtree.minX + street.rtree.maxX) / 2 BETWEEN $topLeftLon AND $bottomRightLon 
+  AND (street.rtree.minY + street.rtree.maxY) / 2 BETWEEN $bottomRightLat AND $topLeftLat )
```

Comment:
Some questions about the math here:

- Given the values `minX=-1, maxX=+1`, what happens when dividing 0.0 by 2?
- Can the brackets be simplified? maybe `(math) BETWEEN x AND y`, the `WHERE` and `AND` condition can be independent.
- The method is named `WITHIN`, but this seems to be an `INTERSECTS`? ie. the text says `find all streets which have a bbox which envelops the specified point` but the input is a box, not a point, the result will contain geometries which 'leak' out of the bounding box? maybe rename the API for clarity?